### PR TITLE
V 3/wui network image

### PR DESCRIPTION
**Pull request by svenvoskamp**
# Changes

- feat: Add `wui-network-image`

### CODE REVIEW

File: packages/ui/src/composites/wui-card-select/index.ts [hunk 1]
```diff
@@ -14,17 +15,31 @@ export class WuiCardSelect extends LitElement {
 
   @property() public name = ''
 
+  @property() public type = 'wallet'
```
Comment (0xAsimetriq) [inline packages/ui/src/composites/wui-card-select/index.ts:18]:
```suggestion
  @property() public type: 'wallet' | 'network' = 'wallet'
```

File: packages/ui/src/composites/wui-card-select/index.ts [hunk 2]
```diff
@@ -14,17 +15,31 @@ export class WuiCardSelect extends LitElement {
 
   @property() public name = ''
```
Comment (0xAsimetriq) [inline packages/ui/src/composites/wui-card-select/index.ts:16]:
```suggestion
  @property() public name = 'unknown'
```

File: packages/ui/src/composites/wui-network-image/index.ts [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+import '../../components/wui-image'
+import { resetStyles } from '../../utils/ThemeUtil'
+import styles from './styles'
+
+@customElement('wui-network-image')
+export class WuiNetworkImage extends LitElement {
+  public static styles = [resetStyles, styles]
+
+  // -- State & Properties -------------------------------- //
+
```
Comment (0xAsimetriq) [inline packages/ui/src/composites/wui-network-image/index.ts:13]:
```suggestion
```

File: packages/ui/src/composites/wui-network-image/index.ts [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+  @property() public networkName = ''
```
Comment (0xAsimetriq) [inline packages/ui/src/composites/wui-network-image/index.ts:14]:
```suggestion
  @property() public name = 'unknown'
```

No need to prefix with network as composite name already indicates that

File: packages/ui/src/composites/wui-network-image/index.ts [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+
+  @property() public src?: string
```
Comment (0xAsimetriq) [inline packages/ui/src/composites/wui-network-image/index.ts:16]: I believe we use imgSrc for similar properties elsewhere, let's stick to that standard here as well

File: packages/ui/src/composites/wui-network-image/index.ts [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+    if (this.src) {
+      return html`<wui-image src=${this.src} alt=${this.networkName}></wui-image>
+        <svg width="48" height="53" fill="none">
+          <path
+            d="M20.041 1.061a7.915 7.915 0 0 1 7.918 0l16.082 9.29A7.922 7.922 0 0 1 48 17.21v18.578c0 2.83-1.51 5.445-3.959 6.86l-16.082 9.29a7.915 7.915 0 0 1-7.918 0l-16.082-9.29A7.922 7.922 0 0 1 0 35.79V17.211c0-2.83 1.51-5.445 3.959-6.86l16.082-9.29Z"
+          />
+        </svg>`
+    }
+
+    return html`<wui-icon size="inherit" color="inherit" name="walletPlaceholder"></wui-icon>`
```
Comment (0xAsimetriq) [inline packages/ui/src/composites/wui-network-image/index.ts:34]: We need different placeholder here for network vs same one as for wallet (if one is not available yet, let's contact design team)

**Review by 0xAsimetriq — CHANGES_REQUESTED** [review]

**Comment by svenvoskamp** [comment]
* Resolved most of the feedback. Asked design team for a network placeholder icon. I'll implement this when I receive this. 
* Also added 'network' option to `wui-card-select-loader`

File: packages/ui/src/composites/wui-card-select-loader/index.ts [hunk 7]
```diff
@@ -1,20 +1,42 @@
...
+
+  private shimmerTemplate() {
+    if (this.type === 'network') {
+      return html` <wui-shimmer
+          type=${this.type}
+          width="48px"
+          height="54px"
+          borderRadius="xs"
+        ></wui-shimmer>
+        <svg width="48" height="53" fill="none">
```
Comment (0xAsimetriq) [inline packages/ui/src/composites/wui-card-select-loader/index.ts:31]: This is the same svg as used in wui-network-image component right, maybe let's move it to svg's folder like with icons and reuse?

File: packages/ui/src/composites/wui-card-select-loader/styles.ts [hunk 8]
```diff
@@ -10,5 +10,30 @@ export default css`
...
     border-radius: var(--wui-border-radius-xs);
+    position: relative;
+  }
+
+  wui-shimmer[type='network'] {
+    border: none;
+    width: 48px;
+    height: 54px;
+    -webkit-clip-path: path(
+      'M20.041 1.061a7.915 7.915 0 0 1 7.918 0l16.082 9.29A7.922 7.922 0 0 1 48 17.21v18.578c0 2.83-1.51 5.445-3.959 6.86l-16.082 9.29a7.915 7.915 0 0 1-7.918 0l-16.082-9.29A7.922 7.922 0 0 1 0 35.79V17.211c0-2.83 1.51-5.445 3.959-6.86l16.082-9.29Z'
```
Comment (0xAsimetriq) [inline packages/ui/src/composites/wui-card-select-loader/styles.ts:21]: Let's make this path a css variable and reuse inside clips

**Pull request merged**